use-cases/financial_payment_classification/financial_payment_classification.ipynb

```diff
@@ -0,0 +1,863 @@
+{
```

Comment:
<p>Can you remove version pin here? Latest is 2.76, doing {sys.executable} -m pip install --upgrade sagemaker should suffice</p><p> </p><p>Do we really need to bump pip?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3017/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
removed the pin